components/main.js

```diff
@@ -63,6 +63,12 @@ exports.main = function (API) {
 
 
 
+		const FIREBUG_API = makeAPI(API, {
+			name: "adapters/firebug"
+		});
+		const FIREBUG_EXPORTS = require("./adapters/firebug").for(FIREBUG_API);
```

Comment:
Require and init the adapter module in our app. This will be abstracted away using a standard PINF module wrapper near term. The wrapper will provide for module-local `console` implementation and allow for APIs to be down-scoped to a module context transparently.